photon-client/src/components/dashboard/tabs/TargetsTab.vue

```diff
@@ -3,12 +3,32 @@ import { useCameraSettingsStore } from "@/stores/settings/CameraSettingsStore";
 import { PipelineType } from "@/types/PipelineTypes";
 import { useStateStore } from "@/stores/StateStore";
 
+const wrapToPi = (delta: number): number => {
```

Comment:
I feel like this should be renamed to `angleModulus` to match the method in MathUtils, I feel like that name is more indicative of what the method does than `wrapToPi`.

Reply:
I think both are fine, but I actually think this is more descriptive? IDK